src/concrete/ml/onnx/ops_impl.py

```diff
@@ -984,11 +1007,19 @@ def numpy_greater_or_equal(
     Args:
         x (numpy.ndarray): Input tensor
         y (numpy.ndarray): Input tensor
+        lsbs_to_remove (Optional[int]): The `AutoRounder` which determines
```

Comment:
better to keep these docstrings coherent with one another (`numpy_greater` has noather docstring here)